userguide/platform/userguide_configuration.adoc

```diff
@@ -29,11 +29,25 @@ boolean isInvoicingSystemEnabled();
 
 == Per tenant properties
 
+Some of our configuration properties can be specified at the tenant level, i.e they can differ for each tenant that specified its own properties. Such properties will have an `InternalTenantContext` in the signature of the method. Example:
```

Comment:
👍 but I'm also wondering if we should instead have a table and just list / explain them all (that way, users don't have to go back to the code)?